components/Resources.js

```diff
@@ -18,6 +18,14 @@ const resourceCategories = [
         name: "Developer's Guide to Content Creation",
         url: "https://www.stephaniemorillo.co/product-page/the-developer-s-guide-to-content-creation",
       },
+      {
+        name: "Developer Relations: How to Build and Grow a Successful Developer Program",
+        url: "https://www.devrelbook.com/",
```

Comment:
This is already added above, correct?

Reply:
I don't see this above 🤔 this is the only time the book appears in the resources list

Reply:
It's called the DevRel handbook above. Check out the Preview link and you'll see it under resources: https://whatisdevrel-git-fork-fgiuliani-add-resources-randomteam.vercel.app/

Reply:
@Nutlope they are two different books

Reply:
Oops you're right lol. Can we shorten the name? I'd like the name to be as long or shorter than the other books in the resources section

Reply:
@Nutlope how about now? Don't know if the idea is not showing the authors' name. "Developer Relations" sounds pretty generic 😅

Reply:
Would rather not exceed this line and yeah I didn't put author names for the other ones so I don't think we need then. I agree that sounds generic but I don't see a better way to stay in the current design

![CleanShot 2022-09-20 at 12 22 29@2x](https://user-images.githubusercontent.com/63742054/191312264-95ca62c1-418a-4cac-9506-749555941d81.png)

Reply:
@Nutlope authors removed